server/handlers/tunnel_test.go

```diff
@@ -14,14 +19,32 @@ import (
 	utils "github.com/globe-and-citizen/layer8-utils"
 )
 
+func makeInitTunnelRequest(clientBackendUrl string) *http.Request {
```

Comment:
the function name is misleading, its contains logic of preparing test data but you only name it  `makeInitTunnelRequest` try to put other name or change the content of the function